scripts/debug-world.lua

```diff
@@ -2,7 +2,7 @@ local util = require("__EditorExtensions__/scripts/util")
 
 --- @param surface LuaSurface
 local function setup_surface(surface)
-  surface.generate_with_lab_tiles = true
+  surface.generate_with_lab_tiles = settings.global["ee-generate-debug-world-with-lab-tiles"].value --[[@as boolean]]
```

Comment:
If lab tiles are disabled, then the surface doesn't need to be cleared, and I would also keep the clouds turned on.